### DISCUSSION TITLE
安装的时候遇到这个，请问怎么处理

### DISCUSSION THREAD

**Discussion by proteincontent** [amll-dev/applemusic-like-lyrics#238] [Q&A] [answered]
![PixPin_2024-03-09_12-21-13](https://github.com/Steve-xmh/applemusic-like-lyrics/assets/161989788/38e5c933-1d18-4372-a6cc-1e7c7915abfe)

**Comment by Linho1219** [comment, marked as answer]
不确定你的需求，请完整表述。

- 如果你需要在你的项目中引入 AMLL，请直接从 npm 安装，<amll.dev> 上提供了文档
- 如果你需要在本地使用 AMLL Player，请从 [releases](https://github.com/amll-dev/applemusic-like-lyrics/releases/tag/main-player-dev) 下载构建产物
- 如果你需要在本地构建这几个包，请在安装依赖后使用 `pnpm nx build:libs`
- 如果你需要在本地构建 AMLL Player，请在构建上述包后执行 `pnpm nx build player` 和 `pnpm -F player tauri build`

由于时间久远，现关闭此讨论。如有进一步需求请重新打开或新建条目。
